t/BikeSharing/IndegoPHL.t

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+ddg_spice_test(
+    [qw( DDG::Spice::BikeSharing::IndegoPHL )],
+    'philly indego' => test_spice(
+        '/js/spice/bike_sharing/indego_phl/philly',
+        call_type => 'include',
+        caller => 'DDG::Spice::BikeSharing::IndegoPHL',
+    is_cached => 1,
```

Comment:
@marianosimone please see https://github.com/duckduckgo/zeroclickinfo-spice/pull/2102#commitcomment-13162269 for an explanation of the caching params.

In these cases you want `is_cached` on, because the query always maps to the same API call.

For the CitiBike Spice, we should probably have it off when the query is just `citibike` because we use the user's city to determine the API call. Thus if someone in Queens or Brooklyn search "citibike", the API call will use their city to specify the burrough, and so `is_cached` should be disabled, meaning we always run the `handle()` sub when the IA triggers.